tilings/strategy_pack.py

```diff
@@ -276,9 +276,18 @@ def insertion_row_and_col_placements(
 
     @classmethod
     def only_root_placements(
-        cls, length: int = 3, max_num_req: Optional[int] = 1,
+        cls,
+        length: int = 3,
+        max_num_req: Optional[int] = 1,
+        max_placement_rules_per_req: Optional[int] = None,
     ) -> "TileScopePack":
-        name = "only_length_{}_root_placements".format(length)
+        if max_num_req is not None:
+            name = "only_length_{length}_{max_num_req}_reqs_root_placements"
+        else:
+            name = "only_length_{length}_root_placements"
```

Comment:
Do we want `max_placement_rules_per_req` in the name too?

Reply:
I thought about that but it made the name too long in my opinion. We still have all the details of the pack that we ran as we save the JSON of the pack in our database